samples/cpp/speech_sample/main.cpp

```diff
@@ -221,14 +224,18 @@ int main(int argc, char* argv[]) {
             executableNet = core.compile_model(model, deviceStr, genericPluginConfig);
         } else {
             slog::info << "Importing model to the device" << slog::endl;
-            std::istringstream streamrq(FLAGS_rg);
+            std::ifstream streamrq(FLAGS_rg, std::ios_base::binary | std::ios_base::in);
+            if (!streamrq.is_open()) {
+                throw std::runtime_error("Cannot open model file " + FLAGS_rg);
+            }
             executableNet = core.import_model(streamrq, deviceStr, genericPluginConfig);
+            streamrq.close();
```

Comment:
We don't need to close ifstream manually

Reply:
ok